packages/crypto/lib/models/block.js

```diff
@@ -374,15 +378,20 @@ module.exports = class Block {
    * @static
    */
   static serializeFull (block) {
+    const serializedBlock = Block.serialize(block, true);
+    const transactions = block.transactions
+
     const buf = new ByteBuffer(1024, true)
```

Comment:
The size of `1024` seems to cause issues on mainnet because of the size of the genesis block while it doesn't happen on testnet or devnet.

`new ByteBuffer(serializedBlock.length + transactions.length * 4, true)` seems to resolve the mainnet issue.

Reply:
Results of logging `serializedBlock.length + transactions.length * 4` for the genesis block.

```
mainnet = 6156
devnet = 395
testnet = 799
```